.github/workflows/ci.yml

```diff
@@ -15,3 +15,4 @@ jobs:
     uses: fastify/workflows/.github/workflows/plugins-ci.yml@v3
     with:
       license-check: true
+      lint: true
```

Comment:
we should update the package.json test script as well

Reply:
like this?